engine/src/flutter/shell/platform/embedder/embedder.h

```diff
@@ -1629,6 +1629,8 @@ typedef struct {
   double elevation;
   /// Describes how much space the semantics node takes up along the z-axis.
   double thickness;
+  /// A technical identifier for the node.
```

Comment:
Consider copying more of the docs from `Semantics.identifier`: https://api.flutter.dev/flutter/semantics/SemanticsNode/identifier.html

```suggestion
  /// An identifier for the semantics node in native accessibility hierarchy.
  ///
  /// This value should not be exposed to the users of the app.
  ///
  /// This is usually used for UI testing with tools that work by querying the
  /// native accessibility, like UIAutomator, XCUITest, or Appium.
```

Reply:
Done.